frontend/src/components/PushPopover.vue

```diff
@@ -122,6 +122,7 @@ async function applyCta(kind: CallToActionKind) {
   let action: PushActionDTO = (await pushStore.applyPush(push.value!.kind, kind)).action;
   switch (kind) {
     case CallToActionKind.CreateSlice:
+      await catalogStore.loadCatalog(projectStore.currentProjectId ?? 0);
```

Comment:
Should not have been call or raise an error if currentProjectId is not defined, no ?
Or redirect ?